dlk/backends/src/intel_hls/qgemm.cpp

```diff
@@ -48,9 +48,9 @@ hls_avalon_slave_component void intel_hls_qgemm_impl(
 
     for (unsigned ar = 0; ar < a_row; ar++) {
 #pragma unroll 8
-      hls_register int16 y0[p::num_pe] = {0, 0, 0, 0, 0, 0, 0, 0};
+      hls_register int16 y0[p::num_pe] = {0, 0, 0, 0};
 
-      hls_register int16 y1[p::num_pe] = {0, 0, 0, 0, 0, 0, 0, 0};
+      hls_register int16 y1[p::num_pe] = {0, 0, 0, 0};
```

Comment:
Do we use `qgemm` for something btw? :thinking: 
If not, is not it better to remove it from the repo to decrease everyone confusion?

Reply:
we can delete them.
I just kept it because I thought I was gonna remove unnecessary files later after it merged otherwise many unrelated modifications appeared in the pull request.

Reply:
yep, I would prefer to keep the amount of changes per PR small too...

it is fine to do it later.